test/test_files/abl_godunov_forcetimetable/abl_godunov_forcetimetable.inp

```diff
@@ -0,0 +1,91 @@
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#            SIMULATION STOP            #
+#.......................................#
+time.stop_time               =   5.0     # Max (simulated) time to evolve
+time.max_step                =   10          # Max number of time steps
+
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#         TIME STEP COMPUTATION         #
+#.......................................#
+time.fixed_dt         =   0.4        # Use this constant dt if > 0
+time.cfl              =   0.95         # CFL factor
+
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#            INPUT AND OUTPUT           #
+#.......................................#
+io.restart_file = "../abl_godunov_timetable/chk00005"
+time.plot_interval            =  10       # Steps between plot files
+time.checkpoint_interval      =  5       # Steps between checkpoint files
+
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#               PHYSICS                 #
+#.......................................#
+incflo.gravity          =   0.  0. -9.81  # Gravitational force (3D)
+incflo.density             = 1.0          # Reference density 
+
+incflo.use_godunov = 1
+incflo.diffusion_type = 2
+transport.viscosity = 1.0e-5
+transport.laminar_prandtl = 0.7
+transport.turbulent_prandtl = 0.3333
+turbulence.model = Smagorinsky
+Smagorinsky_coeffs.Cs = 0.135
+
+
+incflo.physics = ABL
+ICNS.source_terms = BoussinesqBuoyancy CoriolisForcing BodyForce
+BoussinesqBuoyancy.reference_temperature = 300.0
+ABL.reference_temperature = 300.0
+CoriolisForcing.latitude = 41.3
+
+# ABL forcing is not turned on, but ABL Field Init needs velocity values
+# This is required in the absence of incflo.velocity
+ABLForcing.velocity_timetable = "../abl_godunov_timetable/time_table.txt"
+# Note: the use of forcing timetable does not depend on velocity timetable;
+# however, if a precursor uses velocity timetable, force timetable is
+# recommended for subsequent simulations
+BodyForce.type = uniform_timetable
+BodyForce.uniform_timetable_file = "../abl_godunov_timetable/abl_forces.txt"
```

Comment:
Should we include an example file here instead of relying on the output from the other regtest? Or is the plan to do that in the documentation?

Reply:
I see this as the main utility of the new feature, so that's why I set it up with a precursor. But yeah, there should be documentation.